### PR TITLE
Job description alt

### PR DESCRIPTION
**Pull request by Meghan-Rossi**
Fixed version of https://github.com/PolarisSS13/Polaris/pull/5489 .  
Including
* Resolved merge conflicts
* Spelling and grammar corrections
* Added missing alt-title datum for Research Director

### CODE REVIEW

**Comment by TheFurryFeline** [comment]
Nice to see someone's taking that project up after it being stalled for over a year.

File: code/modules/client/preference_setup/occupation/occupation.dm [hunk 1]
```diff
@@ -166,6 +168,45 @@
...
+			if(job.head_position)
+				dat += "You manage this department."
+
+		dat += "You answer to <b>[job.supervisors]</b> normally."
+
+		dat += "<hr style='clear:left;'>"
+		if(config.wikiurl)
+			dat += "<a href='?src=\ref[src];job_wiki=[rank]'>Open wiki page in browser</a>"
+
+		var/alt_title = pref.player_alt_titles[job.title]
```
Comment (Leshana) [inline code/modules/client/preference_setup/occupation/occupation.dm]: Both for consistency with other code above, and best practice to use the accessor proc.
```suggestion
		var/alt_title = pref.GetPlayerAltTitle(job)
```

File: code/modules/client/preference_setup/occupation/occupation.dm [hunk 2]
```diff
@@ -82,8 +82,10 @@
 			index = 0
 
 		. += "<tr bgcolor='[job.selection_color]'><td width='60%' align='right'>"
+
 		var/rank = job.title
 		lastJob = job
+		. += "<a href='?src=\ref[src];job_info=[rank]'>"
```
Comment (Leshana) [inline code/modules/client/preference_setup/occupation/occupation.dm]: You've added an opening `<a>` tag here.  May want to consider adding a matching closing `</a>` tag after the rank everywhere below so it generates valid HTML.

File: code/game/jobs/job/medical.dm [hunk 3]
```diff
@@ -53,18 +103,30 @@
...
 	selection_color = "#013D3B"
 	economic_modifier = 5
 	access = list(access_medical, access_medical_equip, access_morgue, access_surgery, access_chemistry, access_virology, access_genetics)
 	minimal_access = list(access_medical, access_medical_equip, access_chemistry)
-	alt_titles = list("Pharmacist")
-
 	minimal_player_age = 3
 
 	outfit_type = /decl/hierarchy/outfit/job/medical/chemist
+	job_description = "A Chemist produces and maintains a stock of basic to advanced chemicals for medical and occasionally research use. \
```
Comment (Atermonera) [inline code/game/jobs/job/medical.dm]: `[...] stocks of various chemicals ranging in complexity for medical [...]`

**Comment by Atermonera** [comment]
And since it's been well over a year since this stuff was last tested, have you tested this yourself?

**Comment by TheFurryFeline** [comment]
https://github.com/PolarisSS13/Polaris/pull/6762 seems to have made this conflict.

**Comment by Atermonera** [comment]
Restarted travis in case they fell over, complaining about
old vars

**Pull request merged**